package.json

```diff
@@ -34,6 +34,7 @@
     "@nx/angular": "19.1.0",
     "@swc/helpers": "0.5.3",
     "@tanstack/angular-query-experimental": "^5.12.1",
+    "node": "^18.18.2",
```

Comment:
why did you need to add node ?

Reply:
I forgot to remove it since i had accidentally installed it